extensions/observability-devservices/common/src/main/java/io/quarkus/observability/common/ContainerConstants.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.observability.common;
+
+public final class ContainerConstants {
+
+    // Images
+
+    public static final String LGTM = "docker.io/grafana/otel-lgtm:0.3.0";
```

Comment:
I just pushed `grafana/otel-lgtm:0.4.0` to Docker hub. No major changes, mostly version updates, but maybe you want to update to be on the latest version.

Reply:
Updated, tests still pass, so all OK, tnx. :-)